x-pack/test/rule_registry/spaces_only/tests/trial/get_summarized_alerts.ts

```diff
@@ -192,13 +193,11 @@ export default function createGetSummarizedAlertsTest({ getService }: FtrProvide
         executionId: execution1Uuid,
       });
 
-      // Refresh the index so the data is available for reading
-      await es.indices.refresh({ index: `${ruleDataClient.indexName}*` });
-
```

Comment:
No longer need to wait for refresh as it's done automatically in the rule registry now => https://github.com/elastic/kibana/pull/147360/files#diff-cb5b88a0f28b67a629a54e0101adf980bd20b90812e042f65e59082316cc1570R346

Reply:
There are 2 more of it :) 
lines 217 and 239

Reply:
Good catch! Fixed in https://github.com/elastic/kibana/pull/147664/commits/232e81c03380fd323184518f9da2f2f8b6c6620a.